src/mlpack/bindings/python/wrapper_functions_impl.hpp

```diff
@@ -35,9 +35,12 @@ inline std::string GetValidName(const std::string& paramName)
 {
   std::string correctParamName = paramName;
 
-  if(paramName == "lambda") correctParamName = "lambda_";
-  else if(paramName == "input") correctParamName = "input_";
-  else correctParamName = paramName;
+  if (paramName == "lambda")
+    correctParamName = "lambda_";
+  else if (paramName == "input")
+    correctParamName = "input_";
+  else
+    correctParamName = paramName;
```

Comment:
The last `else` path seems redundant, as `correctParamName` is already set to `paramName` at the start of the function.

Alternatively, change the declaration at the start to `std::string correctParamName;`. This would increase efficiency, as `correctParamName` is overwritten in all `if` and `else` code paths.

Reply:
Ah, good point, thanks!  Fixed in 0bf61532eb.